Applications/Layers/jni/meson.build

```diff
@@ -1,8 +1,15 @@
-build_root = meson.build_root()
-res_path = meson.current_source_dir() / '..' / 'res'
+fs = import('fs')
```

Comment:
Please move closer to begin of file, it'll be useful
It's include/import as in other language.